src/library/Uno.Material/Styles/Controls/v2/CalendarDatePicker.xaml

```diff
@@ -279,6 +279,7 @@
                                    Height="2" />
 
                         <Grid Margin="{ThemeResource CalendarDatePickerContentMargin}"
+                              MinWidth="76"
```

Comment:
i think the problem might stems from the calendar having a fixed height
and that the TextBlock.TextWrap is set to wrap

can you change it to NoWrap and see without the min-width, the control would adjust itself?

Reply:
I just retested this issue and it is fixed on latest master, the date on Skia is no longer in YYYY-MM-DD format, it is now in MM-DD-YYYY format like on all the other platforms.

I'm not sure what fixed it, update of uno? Lightweight styling? I can close this PR anyway as it isn't needed anymore